### PR TITLE
Minify item scripts

### PR DESCRIPTION
**Pull request by secretdataz**
This PR minifies item scripts by removing usage of ternary operator (`?:`) and replace it with `min` and `max` command everywhere possible to improve readability. Every scripts still behave the same.

### CODE REVIEW

**Comment by Lemongrass3110** [comment]
@rathena/rathena-script-db-developers could someone please have a look at this one?

**Comment by Lemongrass3110** [comment]
@secretdataz 
https://travis-ci.org/rathena/rathena/jobs/120602008#L1446

File: db/re/item_db.txt [hunk 1]
```diff
@@ -2768,7 +2768,7 @@
...
 4660,Scout_Basilisk_Card,Scout Basilisk Card,6,20,,10,,,,,,,,32,,,,,{ bonus2 bSubSize,Size_Small,5; bonus2 bSubSize,Size_Medium,5; },{},{}
 4661,Charge_Basilisk_Card,Charge Basilisk Card,6,20,,10,,,,,,,,32,,,,,{ bonus2 bSubSize,Size_Medium,20; bonus2 bSubSize,Size_Large,20; bonus2 bSubSize,Size_Small,-15; },{},{}
-4662,Big_Eggring_Card,Big Eggring Card,6,20,,10,,,,,,,,4,,,,,{ bonus bAtk,25-5*((readparam(bStr) > 50) ? 5 : readparam(bStr)/10); bonus bMatk,25-5*((readparam(bInt) > 50) ? 5 : readparam(bInt)/10); bonus bAspdRate,10-2*((readparam(bAgi) > 50) ? 5 : readparam(bAgi)/10); bonus bMaxHP,1000-200*((readparam(bVit) > 50) ? 5 : readparam(bVit)/10); bonus bLongAtkRate,5-1*((readparam(bDex) > 50) ? 5 : readparam(bDex)/10); bonus bCritAtkRate,10-2*((readparam(bLuk) > 50) ? 5 : readparam(bLuk)/10); },{},{}
+4662,Big_Eggring_Card,Big Eggring Card,6,20,,10,,,,,,,,4,,,,,{ bonus bAtk,25-5*max(readParam(bStr)/10,5); bonus bMatk,25-5*max(readParam(bInt)/10,5); bonus bAspdRate,10-2*max(readParam(bAgi)/10,5); bonus bMaxHP,1000-200*max(readParam(bVit)/10,5); bonus bLongAtkRate,5-1*max(readParam(bDex)/10,5); bonus bCritAtkRate,10-2*max(readParam(bLuk)/10,5); },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: 5 is the max.
It should be min function for this item

File: db/re/item_db.txt [hunk 2]
```diff
@@ -2768,7 +2768,7 @@
 4659,Eggring_Card,Eggring Card,6,20,,10,,,,,,,,16,,,,,{ bonus bLuk,2; bonus bMaxHP,50; },{},{}
 4660,Scout_Basilisk_Card,Scout Basilisk Card,6,20,,10,,,,,,,,32,,,,,{ bonus2 bSubSize,Size_Small,5; bonus2 bSubSize,Size_Medium,5; },{},{}
 4661,Charge_Basilisk_Card,Charge Basilisk Card,6,20,,10,,,,,,,,32,,,,,{ bonus2 bSubSize,Size_Medium,20; bonus2 bSubSize,Size_Large,20; bonus2 bSubSize,Size_Small,-15; },{},{}
-4662,Big_Eggring_Card,Big Eggring Card,6,20,,10,,,,,,,,4,,,,,{ bonus bAtk,25-5*max(readParam(bStr)/10,5); bonus bMatk,25-5*max(readParam(bInt)/10,5); bonus bAspdRate,10-2*max(readParam(bAgi)/10,5); bonus bMaxHP,1000-200*max(readParam(bVit)/10,5); bonus bLongAtkRate,5-1*max(readParam(bDex)/10,5); bonus bCritAtkRate,10-2*max(readParam(bLuk)/10,5); },{},{}
+4662,Big_Eggring_Card,Big Eggring Card,6,20,,10,,,,,,,,4,,,,,{ bonus bAtk,25-5*min(readParam(bStr)/10,5); bonus bMatk,25-5*min(readParam(bInt)/10,5); bonus bAspdRate,10-2*min(readParam(bAgi)/10,5); bonus bminHP,1000-200*min(readParam(bVit)/10,5); bonus bLongAtkRate,5-1*min(readParam(bDex)/10,5); bonus bCritAtkRate,10-2*min(readParam(bLuk)/10,5); },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: Blame replace ? :p
bminHP -> bMaxHP

Comment (secretdataz) [inline db/re/item_db.txt]: lol
Should've used max( to min(

**Comment by Atemo** [comment]
Do you plan to add another commit ?
I didn't saw any other mistake, good to merge for me

**Comment by secretdataz** [comment]
That's all for now.

**Pull request merged**